pkg/grpc/config/config.proto

```diff
@@ -159,7 +159,7 @@ message Settings {
   google.protobuf.Duration grpc_server_max_connection_age = 48;
   google.protobuf.Duration grpc_server_max_connection_age_grace = 49;
   optional string forward_auth_url = 50;
-  optional string databroker_service_url = 52;
+  repeated string databroker_service_urls = 52;
```

Comment:
just double checking but it is safe to change to repeated without breaking change right?

Reply:
Saw referenced thread about this very topic :); ignore 👍

Reply:
> Based on my understanding of protobuf it is safe to add repeated to the existing to field since it's not packed because its a string. In the encoded serialization format each field is prefixed with its number, so repeated fields simply re-use the same field number multiple times. Parsers are supposed to tolerate this and use the last value seen for the field according to https://developers.google.com/protocol-buffers/docs/encoding#optional